pkg/operator/edgefs/cluster/target/pod.go

```diff
@@ -183,10 +184,14 @@ func (c *Cluster) makeDaemonContainer(containerImage string, dro DevicesResurrec
 		},
 	}
 
+	// get cluster wide sync option, and apply for deploymentConfig
+	clusterStorageConfig := config.ToStoreConfig(c.Storage.Config)
+	logger.Infof("clusterStorageConfig %+v", clusterStorageConfig)
```

Comment:
From my perspective this should be `Debugf` and not be logged as info level, even though well yes it is information for the user but none the user needs to know at "all" points the function/method runs, right?

Reply:
Yes. Done!